libvirt/resource_libvirt_volume.go

```diff
@@ -90,8 +90,8 @@ func resourceLibvirtVolumeCreate(d *schema.ResourceData, meta interface{}) error
 		poolName = d.Get("pool").(string)
 	}
 
-	PoolSync.AcquireLock(poolName)
-	defer PoolSync.ReleaseLock(poolName)
+	mutexKV.Lock(poolName)
```

Comment:
A really small comment: instead of `mutexKV`, I would use a name that references the pool, `poolMutexKV` or something like that, so it clearly states that this is about the _pool_...

Reply:
ok thx i will do it :sunflower: